.tekton/basic-component-pipeline.yaml

```diff
@@ -105,7 +105,7 @@ spec:
     value: $(tasks.clone-repository.results.commit)
   - description: ""
     name: JAVA_COMMUNITY_DEPENDENCIES
-    value: $(tasks.build-container.results.JAVA_COMMUNITY_DEPENDENCIES)
+    value: $(tasks.build-container-amd64.results.JAVA_COMMUNITY_DEPENDENCIES)
 
```

Comment:
[not about this line] Please adjust comment in the `init` task similar to https://github.com/stackrox/scanner/pull/1574/files#diff-d2c759f3ab7490911f1c024eed8612d5faa9e9a2b2ca8e26b063ab3c2791947cR125 (because there's no `apply-tags` thing any more).

Reply:
This is still open. It's about the current line 120.

```yaml
# expected input. We later actually add this tag on a built image with apply-tags task.
```
should be
```yaml
# expected input. We later actually add this tag on a built image with build-image-manifest-konflux task.
```